src/libsync/vfs/cfapi/cfapiwrapper.h

```diff
@@ -28,7 +28,7 @@ class VfsCfApi;
 namespace CfApiWrapper
 {
 
-class OWNCLOUDSYNC_EXPORT ConnectionKey
+class NEXTCLOUD_CFAPI_EXPORT ConnectionKey
```

Comment:
@mgallien I am having a hard time understanding the reason behind having one more custom define for `__declspec`. Why adding a duplicate with a custom name `NEXTCLOUD_CFAPI_EXPORT`? If we want to get rid of `OWNCLOUDSYNC_EXPORT`, why not rename it to `NEXTCLOUDSYNC_EXPORT` in a separate PR, and replace all its usage places?

Reply:
it is not possible to use the same because we are building another module and in that case, the macro is expanded to an import instead of an export and compilation will fail
in that case, we import from libsync library and we export from cfapi plugin library (needed for the automated tests)
it is cleaner to use different ones such